tests/checkBoxes.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { test, expect } from '@playwright/test';
+import exp from 'constants';
+
+test.beforeEach(async({page}) => {
+    await page.goto('https://petclinic.bondaracademy.com/welcome')
+})
+
+test.describe('Checkboxes', () => {
+    test.beforeEach(async ({page}) => {
+        await page.getByRole('button', {name: " Veterinarians"}).click()
+        await page.getByRole('link', {name: " All"}).click() //1. Select the VETERINARIANS menu item in the nav bar, then select "All"
+    })
+
+    test('validate selected specialties', async ({page}) => {
+        const vetDoctorTableHeader = page.locator('app-vet-list h2')
+        await expect(vetDoctorTableHeader).toHaveText('Veterinarians') //2. Assert Veterinarians text is displayed above table
```

Comment:
lines 15 and 16 could be a single line since `vetDoctorTableHeader` was not used anywhere else in the code

Reply:
I understand what you mean by limiting lines of code, but I do not see how these two lines can be combined. `toHaveText` method does not exist on a locator. And I can't create the assertion when combining.

Reply:
`await expect(page.locator('app-vet-list h2')).toHaveText('Veterinarians')`

Reply:
Hahaha, it would not work for me before because I did not wrap the `page.locator` in parenthesis.